experiments/functional/jiva-volume-resize/test.yml

```diff
@@ -0,0 +1,334 @@
+            --no-headers -o custom-columns=:.metadata.name
+          args:
+            executable: /bin/bash
+          register: replica_pod
+          failed_when: "replica_pod.rc != 0"
+
+        - name: Restart all the replica pods of jiva volume
+          shell: >
+            kubectl delete pod "{{ item }}" -n {{ app_ns }}  
+          args:
+            executable: /bin/bash
+          with_items: "{{ replica_pod.stdout_lines }}"
+
+        - name: Checking for all replicas to be in Running state
+          shell: >
+            kubectl get pod -n {{ app_ns }} -l openebs.io/replica=jiva-replica 
+            --no-headers -o custom-columns=:status.phase
+          register: running_rep_status
+          until: "((running_rep_status.stdout_lines|unique)|length) == 1 and 'Running' in running_rep_status.stdout"
+          delay: 10
+          retries: 30
+
+        - name: Obtain the iSCSI Target IP (controller service IP address)
+          shell: >
+            kubectl get svc -n {{ app_ns }} -l openebs.io/controller-service=jiva-controller-svc 
+            --no-headers -o custom-columns=:.spec.clusterIP
+          args:
+            executable: /bin/bash
+          register: target_IP
+          failed_when: "target_IP.rc != 0"
+
+        - name: Login to iSCSI Target
+          shell: >
+            kubectl exec -ti "{{ ds_pod.stdout }}" -- bash -c "iscsiadm -m discovery -t st -p "{{ target_IP.stdout }}":3260 -l"
+          args:
+            executable: /bin/bash
+          register: status
+          failed_when: "status.rc != 0"
+
+        - name: Obtain the remounted disk
+          shell: >
+            kubectl exec -ti "{{ ds_pod.stdout }}" -- bash -c "iscsiadm -m session -P3" | grep "{{ app_pv.stdout }}"  -A50 | egrep 'iqn|disk' | grep disk | awk {'print $4'}
+          args:
+            executable: /bin/bash
+          register: remount_disk
+          until: "remount_disk.stdout != ''"
+          delay: 2
+          retries: 30
+
+        - name: Check the file system consistency
+          shell: >
+            kubectl exec -ti "{{ ds_pod.stdout }}" 
+            -- bash -c "e2fsck -f /dev/'{{ remount_disk.stdout }}' -y"
+          args:
+            executable: /bin/bash
+          register: status
+          until: "status.rc == 0"
+          delay: 2
+          retries: 30
+          ignore_errors: yes
+
+        - name: Expand the file system
+          shell: >
+            kubectl exec -ti "{{ ds_pod.stdout }}" -- bash -c "resize2fs /dev/'{{ remount_disk.stdout }}'"
+          args:
+            executable: /bin/bash
+          register: status
+          failed_when: "status.rc != 0"
+
+        - name: Mount the file system
+          shell: >
+            kubectl exec -ti "{{ ds_pod.stdout }}" 
+            -- bash -c "mount /dev/'{{ remount_disk.stdout }}' '{{ item }}'"
+          args:
+            executable: /bin/bash
+          with_items: "{{ mountpath.stdout_lines }}"
+          ignore_errors: yes
```

Comment:
why are we ignoring errors here

Reply:
It may try to mount same mount path which may lead to errors.